hrjobcontract/css/hrjc.css

```diff
@@ -61,6 +61,7 @@
   /* add generic styling for the editor */
   /* add the styling for the awesomness of the resizer */
   /* copy the popover code from bootstrap so this will work even without it */
+  /* stylelint-disable selector-no-qualifying-type */
```

Comment:
why is this necessary? The .css file should not be even considered by the linter.

Reply:
This is not added manually, after build process it is simply copied.